idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -354,12 +357,18 @@ def unmapped_files_in(some_dir):
         for part_idx in range(num_parts):
             tmp_result_dir = "%s/star-part-%d" % (SCRATCH_DIR, part_idx)
             genome_part = "%s/part-%d" % (genome_dir, part_idx)
-            run_star_part(tmp_result_dir, genome_part, unmapped)
-            unmapped = sync_pairs(unmapped_files_in(tmp_result_dir))
+            count_genes = (part_idx == 0)
```

Comment:
actually there's a way to look into if the gtf is being indexed. 

sjdbList.fromGTF.out.tab should exist under the index directory